packages/core/src/routes/setting.ts

```diff
@@ -0,0 +1,27 @@
+import { Settings } from '@logto/schemas';
+
+import koaGuard from '@/middleware/koa-guard';
+import { getSetting, updateSetting } from '@/queries/setting';
+
+import { AuthedRouter } from './types';
+
+export default function settingRoutes<T extends AuthedRouter>(router: T) {
+  router.get('/settings', async (ctx, next) => {
+    ctx.body = await getSetting();
+
+    return next();
+  });
+
+  router.patch(
+    '/settings',
+    koaGuard({
+      body: Settings.guard.pick({ id: true }).merge(Settings.guard.partial()),
+    }),
+    async (ctx, next) => {
+      const { body: setting } = ctx.guard;
```

Comment:
`id` will NOT in body, we need to find-or-create here

Reply:
@darcyYe why this is resolved?

Reply:
if you mean we'll seed settings table, please add an issue in the related project and leave a comment for this

Reply:
Please refer LOG-1294.